src/main/kotlin/io/snyk/plugin/Utils.kt

```diff
@@ -193,7 +194,12 @@
     lateinit var checkIfSastEnabled: () -> Unit
     checkIfSastEnabled = {
         if (settings.sastOnServerEnabled != true) {
-            settings.sastOnServerEnabled = getSnykApiService().getSastSettings()?.sastEnabled ?: false
+            settings.sastOnServerEnabled = try {
+                getSnykApiService().getSastSettings()?.sastEnabled ?: false
+            } catch (t: ClientException) {
```

Comment:
please rename `t` to `ignored` so the exception is marked as intentionally swallowed for the linter